src/test/java/uk/gov/hmcts/darts/common/service/impl/DataAnonymisationServiceImplTest.java

```diff
@@ -419,32 +455,33 @@ void positiveTidyUpTransformedMediaEntities() {
 
         dataAnonymisationService.tidyUpTransformedMediaEntities(userAccount, courtCase);
 
-        verify(dataAnonymisationService, times(1)).expiredMediaRequest(userAccount, hearing1MediaRequestEntity1);
-        verify(dataAnonymisationService, times(1)).expiredMediaRequest(userAccount, hearing1MediaRequestEntity2);
-        verify(dataAnonymisationService, times(1)).expiredMediaRequest(userAccount, hearing1MediaRequestEntity3);
+        verify(dataAnonymisationService).expiredMediaRequest(userAccount, hearing1MediaRequestEntity1);
+        verify(dataAnonymisationService).expiredMediaRequest(userAccount, hearing1MediaRequestEntity2);
+        verify(dataAnonymisationService).expiredMediaRequest(userAccount, hearing1MediaRequestEntity3);
 
 
-        verify(dataAnonymisationService, times(1)).expiredMediaRequest(userAccount, hearing2MediaRequestEntity1);
-        verify(dataAnonymisationService, times(1)).expiredMediaRequest(userAccount, hearing2MediaRequestEntity2);
-        verify(dataAnonymisationService, times(1)).expiredMediaRequest(userAccount, hearing2MediaRequestEntity3);
+        verify(dataAnonymisationService).expiredMediaRequest(userAccount, hearing2MediaRequestEntity1);
+        verify(dataAnonymisationService).expiredMediaRequest(userAccount, hearing2MediaRequestEntity2);
+        verify(dataAnonymisationService).expiredMediaRequest(userAccount, hearing2MediaRequestEntity3);
 
-        verify(dataAnonymisationService, times(1)).expiredMediaRequest(userAccount, hearing3MediaRequestEntity1);
+        verify(dataAnonymisationService).expiredMediaRequest(userAccount, hearing3MediaRequestEntity1);
 
 
-        verify(dataAnonymisationService, times(1))
+        verify(dataAnonymisationService)
             .deleteTransformedMediaEntity(transformedMediaEntity1);
-        verify(dataAnonymisationService, times(1))
+        verify(dataAnonymisationService)
             .deleteTransformedMediaEntity(transformedMediaEntity2);
-        verify(dataAnonymisationService, times(1))
+        verify(dataAnonymisationService)
             .deleteTransformedMediaEntity(transformedMediaEntity3);
-        verify(dataAnonymisationService, times(1))
+        verify(dataAnonymisationService)
             .deleteTransformedMediaEntity(transformedMediaEntity4);
-        verify(dataAnonymisationService, times(1))
+        verify(dataAnonymisationService)
             .deleteTransformedMediaEntity(transformedMediaEntity5);
     }
 
-    @Test
-    void positiveAnonymiseEventByIds() {
+    @ParameterizedTest(name = "Anonymise event by ids with isManuallyRequested = {0}")
+    @ValueSource(booleans = {true, false})
+    void anonymiseEventByIds(boolean isManuallyRequested) {
```

Comment:
Can you amend the name to use the new convention please

Reply:
Updated